meteor-app/imports/api/parts/schema.js

```diff
@@ -1,26 +1,32 @@
 import { Mongo } from 'meteor/mongo'
 import SimpleSchema from  'simpl-schema'
 
-import { createdAt, updatedAt } from '/imports/api/schema'
+import { REGEX_ID, createdAt, updatedAt } from '/imports/api/schema'
 
 const Parts = new Mongo.Collection('parts')
 
 export const PartsSchema = new SimpleSchema({
+  _id: {
```

Comment:
You didn't read my comment properly... import `RegExId` which is basically a predefined type, much like `createdAt`

It's one line of code to use it:

`  _id: RegExId,`